### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by ihaveamac**

### CODE REVIEW

**Comment by TheGoofBall64** [comment]
I'm not a mod. I just thought this absolutely needed pointing out

**Comment by lilyuwuu** [comment]
@TheGoofBall64 um, not sure what you mean there.

**Comment by TheGoofBall64** [comment]
because normally the server mods look at pull requests and request changes. I just kinda typed that out of instinct

**Comment by TheGoofBall64** [comment]
oh come on. i canceled the reviews. give me a second

File: _pages/es_ES/godmode9-usage.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: "Usos de GodMode9"
 
 {% include toc title="Tabla de contenidos" %}
 
-Para soporte (en inglés) para GodMode9, así como ayudar con la creación de scripts y obtener actualizaciones e información, únete al [Discord de GodMode9](https://discord.gg/EGu6Qxw).
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```
Comment (TheGoofBall64) [inline _pages/es_ES/godmode9-usage.txt:7]: unnecessary language change

File: _pages/fr_FR/formatting-sd-(linux).txt [hunk 2]
```diff
@@ -10,7 +10,7 @@ Il s'agit d'une section supplémentaire pour le formatage d'une carte SD afin qu
 
 Si la 3DS reconnaît déjà la carte SD, il n'est pas nécessaire de suivre ce guide.
 
-Cette page est uniquement destinée aux utilisateurs de Linux. Si vous n'êtes pas sur Linux, consultez les pages [Formatage de la carte SD (Windows)](formatting-sd-(Windows)) ou [Formatage de la carte SD (Mac)](formatting-sd-(mac)).
+Cette page est uniquement destinée aux utilisateurs de Linux. If you are not on Linux, check out the [Formatting SD (Windows)](formatting-sd-(windows)) or [Formatting SD (Mac)](formatting-sd-(mac)) pages.
```
Comment (TheGoofBall64) [inline _pages/fr_FR/formatting-sd-(linux).txt:13]: unnecessary language change again

File: _pages/fr_FR/godmode9-usage.txt [hunk 3]
```diff
@@ -4,7 +4,7 @@ title: "Utilisation de GodMode9"
 
 {% include toc title="Table des matières" %}
 
-Pour de l'aide (en anglais) sur GodMode9 ainsi que sur le scripting, et pour être tenu à jour et informé, rejoignez le [Discord GodMode9](https://discord.gg/EGu6Qxw).
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```
Comment (TheGoofBall64) [inline _pages/fr_FR/godmode9-usage.txt:7]: unnecessary language change again

File: _pages/hu_HU/formatting-sd-(linux).txt [hunk 4]
```diff
@@ -10,7 +10,7 @@ Ez egy kiegészítő rész az SD kártya formázásához, hogy az működjön a
 
 Ha a 3DS már felismeri az SD kártyát, ez az útmutató nem szükséges.
 
-Ez az oldal Linux felhasználókra vonatkozik. Ha nem Linux rendszeren vagy, kövesd az [SD formázás (Windows)](formatting-sd-(linux)) vagy [SD formázás (Mac)](formatting-sd-(mac)) útmutatókat.
+Ez az oldal Linux felhasználókra vonatkozik. If you are not on Linux, check out the [Formatting SD (Windows)](formatting-sd-(windows)) or [Formatting SD (Mac)](formatting-sd-(mac)) pages.
```
Comment (TheGoofBall64) [inline _pages/hu_HU/formatting-sd-(linux).txt:13]: unnecessary language change again

File: _pages/hu_HU/godmode9-usage.txt [hunk 5]
```diff
@@ -4,7 +4,7 @@ title: "GodMode9 használata"
 
 {% include toc title="Tartalomjegyzék" %}
 
-Ha segítségre van szükséged GodMode9-nal kapcsolatban (angol nyelven), legyen az scriptelés, vagy a legfrissebb információk beszerzése, csatlakozz a [GodMode9 Discord szerverhez](https://discord.gg/EGu6Qxw).
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```
Comment (TheGoofBall64) [inline _pages/hu_HU/godmode9-usage.txt:7]: unnecessary language change again

File: _pages/it_IT/formatting-sd-(linux).txt [hunk 6]
```diff
@@ -10,7 +10,7 @@ Questa è una sezione aggiuntiva per la formattazione di una scheda SD per il 3D
 
 Se il 3DS riconosce già la scheda SD, questa parte non è necessaria.
 
-Questa pagina è solo per utenti Linux. Se non stai utilizzando Linux, puoi seguire la guida alle pagine [Formattazione SD (Windows)](formatting-sd-(Windows)) o [Formattazione SD (Mac)](formatting-sd-(mac)).
+Questa pagina è solo per utenti Linux. If you are not on Linux, check out the [Formatting SD (Windows)](formatting-sd-(windows)) or [Formatting SD (Mac)](formatting-sd-(mac)) pages.
```
Comment (TheGoofBall64) [inline _pages/it_IT/formatting-sd-(linux).txt:13]: unnecessary language change again

File: _pages/it_IT/godmode9-usage.txt [hunk 7]
```diff
@@ -4,7 +4,7 @@ title: "Usare GodMode9"
 
 {% include toc title="Indice" %}
 
-Per ricevere supporto su GodMode9, aiuto per gli script e informazioni sugli ultimi aggiornamenti, entra nel [server Discord di GodMode9](https://discord.gg/EGu6Qxw) (in Inglese).
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```
Comment (TheGoofBall64) [inline _pages/it_IT/godmode9-usage.txt:7]: unnecessary language change again

File: _pages/ja_JP/formatting-sd-(linux).txt [hunk 8]
```diff
@@ -10,7 +10,7 @@ title: "SDをフォーマット (Linux)"
 
 3DSがすでにSDカードを認識している場合は、この操作は必要ありません。
 
-このページはLinuxユーザー向けです。 Linuxをお使いでない場合は、[Formatting-sd-(Windows)] (formatting-sd-(Windows)) または [Formatting-sd-(Mac)](formatting-sd-(mac)) のページをご覧ください。
+このページはLinuxユーザー向けです。 If you are not on Linux, check out the [Formatting SD (Windows)](formatting-sd-(windows)) or [Formatting SD (Mac)](formatting-sd-(mac)) pages.
```
Comment (TheGoofBall64) [inline _pages/ja_JP/formatting-sd-(linux).txt:13]: unnecessary language change again

File: _pages/ja_JP/godmode9-usage.txt [hunk 9]
```diff
@@ -4,7 +4,7 @@ title: "GodMode9の使用方法"
 
 {% include toc title="目次" %}
 
-GodMode9のサポート(英語)やスクリプト、アップデート、その他の情報につきましては[GodMode9 on Discord](https://discord.gg/EGu6Qxw)にて受け付けております。
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```
Comment (TheGoofBall64) [inline _pages/ja_JP/godmode9-usage.txt:7]: unnecessary language change again

File: _pages/pl_PL/godmode9-usage.txt [hunk 10]
```diff
@@ -4,7 +4,7 @@ title: "Używanie GodMode9"
 
 {% include toc title="Spis Treści" %}
 
-Aby uzyskać pomoc (w języku angielskim) dotyczącą programu GodMode9, a także pomoc w pisaniu skryptów oraz uzyskiwaniu aktualizacji i informacji, dołącz do serwera [GodMode9 Discord](https://discord.gg/EGu6Qxw).
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```
Comment (TheGoofBall64) [inline _pages/pl_PL/godmode9-usage.txt:7]: unnecessary language change again

File: _pages/ro_RO/godmode9-usage.txt [hunk 11]
```diff
@@ -4,7 +4,7 @@ title: "Folosind GodMode9"
 
 {% include toc title="Cuprins" %}
 
-Pentru suport (în engleză) cu GodMode9, precum și ajutor cu scripting şi a obţine actualizări şi informaţii, alăturați-vă [serverului GodMode9 pe Discord](https://discord.gg/EGu6Qxw).
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```
Comment (TheGoofBall64) [inline _pages/ro_RO/godmode9-usage.txt:7]: unnecessary language change again

File: _pages/ru_RU/godmode9-usage.txt [hunk 12]
```diff
@@ -4,7 +4,7 @@ title: "Использование GodMode9"
 
 {% include toc title="Содержание" %}
 
-Для поддержки (на английском языке) по теме GodMode9, а также для помощи в написании скриптов или для получения обновлений, обратитесь на сервер [GodMode9 в Discord](https://discord.gg/EGu6Qxw).
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```
Comment (TheGoofBall64) [inline _pages/ru_RU/godmode9-usage.txt:7]: unnecessary language change again

File: _pages/tr_TR/godmode9-usage.txt [hunk 13]
```diff
@@ -4,7 +4,7 @@ title: "GodMode9 Usage"
 
 {% include toc title = "İçindekiler"%}
 
-Hem GodMode9 hem de script yazımında (İngilizce) destek almak için ve güncellemeler vb. bilgi elde etmek için [GodMode9 Discord](https://discord.gg/EGu6Qxw)'unu ziyaret edin.
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```
Comment (TheGoofBall64) [inline _pages/tr_TR/godmode9-usage.txt:7]: unnecessary language change again

File: _pages/zh_CN/formatting-sd-(linux).txt [hunk 14]
```diff
@@ -10,7 +10,7 @@ title: "格式化 SD 卡（通过 Linux 操作系统）"
 
 如果 3DS 已经识别到了 SD 卡，那么就不需要做这个教程了。
 
-本页教程仅限 Linux 用户。 如果你没有在用 Linux，那么请看看[格式化 SD 卡（通过 Windows 操作系统）](formatting-sd-(windows))或[格式化 SD 卡（通过 Mac 操作系统）](formatting-sd-(mac)) 。
+本页教程仅限 Linux 用户。 If you are not on Linux, check out the [Formatting SD (Windows)](formatting-sd-(windows)) or [Formatting SD (Mac)](formatting-sd-(mac)) pages.
```
Comment (TheGoofBall64) [inline _pages/zh_CN/formatting-sd-(linux).txt:13]: unnecessary language change again

File: _pages/zh_CN/godmode9-usage.txt [hunk 15]
```diff
@@ -4,7 +4,7 @@ title: "GodMode9 使用教程" #
 
 {% include toc title="目录内容" %}
 
-如果你想获得有关 GodMode9 的帮助、帮助编写脚本和获取更新及有关信息（英文），你可以加入 [GodMode9 Discord 服务器](https://discord.gg/EGu6Qxw)。（请注意，如果你身处中国大陆，访问 Discord 可能需要通过科学上网）
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```
Comment (TheGoofBall64) [inline _pages/zh_CN/godmode9-usage.txt:7]: unnecessary language change again

File: _pages/zh_TW/godmode9-usage.txt [hunk 16]
```diff
@@ -4,7 +4,7 @@ title: "如何使用 GodMode9 "
 
 {% include toc title="條目內容" %}
 
-不論寫腳本、獲得更新訊息或更多訊息，您能加入 [GodMode9 的 Discord 伺服器](https://discord.gg/EGu6Qxw) 以獲得更多幫助。
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```
Comment (TheGoofBall64) [inline _pages/zh_TW/godmode9-usage.txt:7]: unnecessary language change again

Did you copy-paste these? If you are going to update the guides other languages, please make sure you don't change the language

**Comment by TheGoofBall64** [comment]
> @TheGoofBall64 um, not sure what you mean there.

Ohhh. I never hit send on the reviews. sorry about that

**Comment by MechanicalDragon0687** [comment]
> > @TheGoofBall64 um, not sure what you mean there.
> 
> Ohhh. I never hit send on the reviews. sorry about that

These updates are automatic as part of the crowdin strings. a change to the line means the line has to be reverified. the discord link changed, so the line must be reverified.

**Comment by TheGoofBall64** [comment]
oh. sorry then

**Pull request merged**